lab-examples/ixiac01/ixiac01.clab.yml

```diff
@@ -6,8 +6,8 @@ topology:
       kind: keysight_ixia-c-one
       image: ghcr.io/open-traffic-generator/ixia-c-one:0.0.1-2755
       exec:
-        - "./ifcfg add eth1 10.1.1.1 24"
-        - "./ifcfg add eth2 10.2.2.1 24"
+        - "./ifcfg add eth1 1.1.1.1 24"
+        - "./ifcfg add eth2 2.2.2.1 24"
```

Comment:
why chanigng 10 for 1?

Reply:
In the file _ipv4_forwarding.go_ test setup is the following, wich is correct, but in the scripts IP addresses are wrong.
Test IPv4 Forwarding with
- Endpoints: OTG **1.1.1.1** -----> 1.1.1.2 DUT **2.2.2.1** ------> OTG 2.2.2.2
- Static Route on DUT: 20.20.20.0/24 -> 2.2.2.2
- TCP flow from OTG: 10.10.10.1 -> 20.20.20.1+